src/services/goToDefinition.ts

```diff
@@ -113,6 +117,18 @@ namespace ts.GoToDefinition {
         }
     }
 
+    function getDefinitionFromOverriddenMember(typeChecker: TypeChecker, node: PropertyName) {
+        const baseDeclaration = findAncestor(node, isClassLike);
+        const baseTypeNode = baseDeclaration ? getEffectiveBaseTypeNode(baseDeclaration) : undefined;
+        const baseType = baseTypeNode ? typeChecker.getTypeAtLocation(baseTypeNode) : undefined;
+        if (baseType === undefined) return;

```

Comment:
It's safe to just `!baseType` because `Type` and `Symbol` are both non-falsely value.